guide/guide/src/main/scala/io/udash/web/guide/views/ext/demo/bootstrap/SimpleFormDemo.scala

```diff
@@ -0,0 +1,110 @@
+package io.udash.web.guide.views.ext.demo.bootstrap
+
+import io.udash._
+import io.udash.bootstrap.button.UdashButton
+import io.udash.bootstrap.form.UdashForm
+import io.udash.css.CssView
+import io.udash.logging.CrossLogging
+import io.udash.web.guide.demos.AutoDemo
+import io.udash.web.guide.styles.partials.GuideStyles
+import scalatags.JsDom
+
+import scala.concurrent.Future
+
+object SimpleFormDemo extends AutoDemo with CrossLogging with CssView {
+
+  import JsDom.all._
+  import io.udash.bootstrap.utils.BootstrapImplicits._
+  import io.udash.web.guide.Context._
+
+  sealed trait ShirtSize
+
+  case object Small extends ShirtSize
+
+  case object Medium extends ShirtSize
+
+  case object Large extends ShirtSize
+
+  trait UserModel {
+    def name: String
+
+    def age: Int
+
+    def shirtSize: ShirtSize
+  }
+
+  object UserModel extends HasModelPropertyCreator[UserModel] {
+    implicit val blank: Blank[UserModel] = Blank.Simple(new UserModel {
+      override def name: String = ""
+
+      override def age: Int = 25
+
+      override def shirtSize: ShirtSize = Medium
+    })
+  }
+
+  private val (rendered, source) = {
+    /*
+    sealed trait ShirtSize
+    case object Small extends ShirtSize
+    case object Medium extends ShirtSize
+    case object Large extends ShirtSize
+
+    trait UserModel {
+      def name: String
+      def age: Int
+      def shirtSize: ShirtSize
+    }
+    object UserModel extends HasModelPropertyCreator[UserModel] {
+      implicit val blank: Blank[UserModel] = Blank.Simple(new UserModel {
+        override def name: String = ""
+        override def age: Int = 25
+        override def shirtSize: ShirtSize = Medium
+      })
+    }
+    */
+
+    def shirtSizeToLabel(size: ShirtSize): String = size match {
```

Comment:
Since UserModel will be used in `ModelProperty` I don't think it's a good idea to add a method to it. I'd rather create extension method to `ModelProperty[UserModel]` that will produce `ReadableProperty[String]` label or leave it as it is.

Reply:
Right, I'd leave this pattern match then.